.eslintrc.js

```diff
@@ -1,211 +1,45 @@
 module.exports = {
   // When adding items to this file please check for effects on sub-directories.
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
     },
-    "sourceType": "module"
+    sourceType: "module",
   },
-  "env": {
-    "node": true
+  env: {
+    node: true,
   },
-  "plugins": [
+  plugins: [
+    "prettier", // "require("eslint-plugin-prettier")
     "import", // require("eslint-plugin-import")
-    "react" // require("eslint-plugin-react")
+    "react", // require("eslint-plugin-react")
   ],
-  "extends": [
-    "eslint:recommended",
-    "plugin:mozilla/recommended" // require("eslint-plugin-mozilla")
+  extends: [
+    "plugin:mozilla/recommended", // require("eslint-plugin-mozilla")
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/react", // require("eslint-plugin-prettier)
+    "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    "plugin:prettier/recommended", // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always
   ],
-  "rules": {
-    "react/jsx-boolean-value": [2, "always"],
-    "react/jsx-closing-bracket-location": [2, "after-props"],
-    "react/jsx-curly-spacing": [2, "never"],
-    "react/jsx-equals-spacing": [2, "never"],
-    "react/jsx-key": 2,
-    "react/jsx-no-bind": 2,
-    "react/jsx-no-comment-textnodes": 2,
-    "react/jsx-no-duplicate-props": 2,
+  rules: {
+    // Why no react/jsx-no-bind?
+    // See: https://twitter.com/Meligy/status/1069074157151780864
+    //      https://reactjs.org/docs/hooks-faq.html#are-hooks-slow-because-of-creating-functions-in-render
+    //      https://cdb.reacttraining.com/react-inline-functions-and-performance-bdff784f5578
+
+    "react/prop-types": 0,
+    // TODO: Remove no-deprecated when finished migrating
+    "react/no-deprecated": 0,
     "react/jsx-no-target-blank": 2,
     "react/jsx-no-undef": 2,
     "react/jsx-pascal-case": 2,
-    "react/jsx-uses-react": 2,
-    "react/jsx-uses-vars": 2,
-    "react/jsx-wrap-multilines": 2,
     "react/no-access-state-in-setstate": 2,
     "react/no-danger": 2,
-    "react/no-deprecated": 0,
     "react/no-did-mount-set-state": 2,
     "react/no-did-update-set-state": 2,
     "react/no-direct-mutation-state": 2,
-    "react/no-is-mounted": 2,
-    "react/no-unknown-property": 2,
-    "react/require-render-return": 2,
-    "react/self-closing-comp": 2,
-
-    "accessor-pairs": [2, {"setWithoutGet": true, "getWithoutSet": false}],
```

Comment:
Intended that all eslint rules removed? Not all are styling related / handled by prettier, but up to you if you want to remove or base the removals on what dmose did for activity stream PR https://github.com/mozilla/activity-stream/pull/5084

Reply:
I tried to remove everything that wasn't part of the default configuration and not super necessary just because keeping this in sync with activity-stream was annoying. I'll take a look though